packages/menu/README.md

```diff
@@ -104,63 +104,67 @@ import { Menu, MenuSeparator, MenuItem } from '@leafygreen-ui/menu';
+| `setOpen`          | `function`                                     | When controlling the component, use `setOpen` to keep track of the `<Menu />` component's state so that clicks on the document's backdrop as well as a user pressing the Escape Key will close the Menu and update the consuming application's local state accordingly. |              |
+| `shouldClose`      | `function`                                     | Determines if the `Menu` should close when the backdrop or Escape keys are clicked. Defaults to true.                                                                                                                                                                   | `() => true` |
+| `align`            | `'top'` \| `'bottom'` \| `'left'` \| `'right'` | Determines the alignment of the `<Menu />` component relative to a reference element, or the element's nearest parent                                                                                                                                                   | `'bottom'`   |
+| `justify`          | `'start'` \| `'middle'` \| `'end'`             | Determines the justification of the `Menu` component (based on the alignment) relative to a reference element or the element's nearest parent                                                                                                                           | `'end'`      |
+| `refEl`            | `node`                                         | Pass a reference to an element that the `Menu` component should be positioned against                                                                                                                                                                                   |              |
+| `trigger`          | `HTMLElement` or `ReactNode`                   | An `HTMLElement` or `ReactNode` against which the Menu will be positioned.                                                                                                                                                                                              |              |
+| `usePortal`        | `boolean`                                      | Will position Menu's children relative to its parent without using a Portal if `usePortal` is set to false. NOTE: The parent element should be CSS position relative, fixed, or absolute if using this option.                                                          | `true`       |
+| `adjustOnMutation` | `boolean`                                      | Determines whether or not the `<Menu/>` should reposition itself based on changes to `trigger` or `referenceElement` position.                                                                                                                                          | `false`      |
```

Comment:
neither `referenceElement` nor `refEl` are defined anywhere in this readme

Reply:
Line 114!

Reply:
ah, then this should at least be updated to be `refEl` or the words "reference element", but not `referenceElement`